### PR TITLE
Add instructions to enable anypoint monitoring

### PR DESCRIPTION
**Pull request by sundippatel**

### CODE REVIEW

**Review by paigehargrave — APPROVED** [review]

**Comment by paigehargrave** [comment]
@sundippatel - PTAL and make sure I didn't introduce any errors here - thank you!

**Comment by sundippatel** [comment]
Looks great, thanks!

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 1]
```diff
@@ -4,11 +4,25 @@ Applications and API gateways deployed to Runtime Fabric include native support
 
 == Anypoint Monitoring
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-monitor-applications.adoc:6]: ORG:
Ensure that the subject of this subtopic is introduced in the text of the topic that contains it, and that it is a peer of all other subtopics at this level.
Ensure also that the text of this subtopic introduces the subtopics that it contains.
GLOBAL org. recommendation for all deliverables.

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 2]
```diff
@@ -4,11 +4,25 @@ Applications and API gateways deployed to Runtime Fabric include native support
...
 
-xref:monitoring::index.adoc[Anypoint Monitoring] provides tools that give feedback from Mule flows and components 
+xref:monitoring::index.adoc[Anypoint Monitoring] includes tools that provide feedback from Mule flows and components 
 in your application network. For applications and API gateways deployed to Runtime Fabric, Anypoint Monitoring 
-provides basic metrics to all customers. Titanium customers have access to additional capabilities including 
+provides basic metrics to all customers. Titanium customers have access to additional capabilities, including 
 application log retrieval and search.
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-monitor-applications.adoc]: Be sure to mention the following contained subtopics, enabling and disabling.

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 3]
```diff
@@ -4,11 +4,25 @@ Applications and API gateways deployed to Runtime Fabric include native support
...
 
+=== Enable Anypoint Monitoring
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-monitor-applications.adoc:9]: In most deliverables, I would expect task topics to not be "buried" at the third level of the information hierarchy.
I think we want "The User Journey" of tasks to be primary.

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 4]
```diff
@@ -4,11 +4,25 @@ Applications and API gateways deployed to Runtime Fabric include native support
 
 == Anypoint Monitoring
 
-xref:monitoring::index.adoc[Anypoint Monitoring] provides tools that give feedback from Mule flows and components 
+xref:monitoring::index.adoc[Anypoint Monitoring] includes tools that provide feedback from Mule flows and components
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-monitor-applications.adoc]:
```suggestion
xref:monitoring::index.adoc[Anypoint Monitoring] includes tools that provide feedback from Mule runtime engine (Mule) flows and components 
```

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 5]
```diff
@@ -4,11 +4,25 @@ Applications and API gateways deployed to Runtime Fabric include native support
 
 == Anypoint Monitoring
 
-xref:monitoring::index.adoc[Anypoint Monitoring] provides tools that give feedback from Mule flows and components 
+xref:monitoring::index.adoc[Anypoint Monitoring] includes tools that provide feedback from Mule flows and components 
 in your application network. For applications and API gateways deployed to Runtime Fabric, Anypoint Monitoring
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-monitor-applications.adoc]:
```suggestion
in your application network. For applications and API gateways deployed to Anypoint Runtime Fabric (Runtime Fabric), Anypoint Monitoring 
```

File: modules/ROOT/pages/manage-monitor-applications.adoc (hunk 2 above)
Comment (kbostrom) [reply modules/ROOT/pages/manage-monitor-applications.adoc]: Be sure to mention the following contained subtopics, enabling and disabling.

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 6]
```diff
@@ -4,11 +4,25 @@ Applications and API gateways deployed to Runtime Fabric include native support
 
 == Anypoint Monitoring
 
-xref:monitoring::index.adoc[Anypoint Monitoring] provides tools that give feedback from Mule flows and components 
+xref:monitoring::index.adoc[Anypoint Monitoring] includes tools that provide feedback from Mule flows and components 
 in your application network. For applications and API gateways deployed to Runtime Fabric, Anypoint Monitoring 
-provides basic metrics to all customers. Titanium customers have access to additional capabilities including 
+provides basic metrics to all customers. Titanium customers have access to additional capabilities, including
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-monitor-applications.adoc]:
```suggestion
provides basic metrics to all customers. Customers subscribed to the Titanium plan have access to additional capabilities, including 
```

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 7]
```diff
@@ -4,11 +4,25 @@ Applications and API gateways deployed to Runtime Fabric include native support
...
-xref:monitoring::index.adoc[Anypoint Monitoring] provides tools that give feedback from Mule flows and components 
+xref:monitoring::index.adoc[Anypoint Monitoring] includes tools that provide feedback from Mule flows and components 
 in your application network. For applications and API gateways deployed to Runtime Fabric, Anypoint Monitoring 
-provides basic metrics to all customers. Titanium customers have access to additional capabilities including 
+provides basic metrics to all customers. Titanium customers have access to additional capabilities, including 
 application log retrieval and search.
 
+=== Enable Anypoint Monitoring
+
+By default, Runtime Fabric is configured to send application metrics to Anypoint Monitoring. Titanium customers also have application logs automatically sent to Anypoint Monitoring by default. In order for metrics and logs to be sent to Anypoint Monitoring, outbound network connectivity must be available.
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-monitor-applications.adoc]:
```suggestion
By default, Runtime Fabric is configured to send application metrics to Anypoint Monitoring. Titanium-plan customers' application logs are automatically sent to Anypoint Monitoring by default. For metrics and logs to be sent to Anypoint Monitoring, outbound network connectivity must be available. 
```

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 8]
```diff
@@ -4,11 +4,25 @@ Applications and API gateways deployed to Runtime Fabric include native support
...
 in your application network. For applications and API gateways deployed to Runtime Fabric, Anypoint Monitoring 
-provides basic metrics to all customers. Titanium customers have access to additional capabilities including 
+provides basic metrics to all customers. Titanium customers have access to additional capabilities, including 
 application log retrieval and search.
 
+=== Enable Anypoint Monitoring
+
+By default, Runtime Fabric is configured to send application metrics to Anypoint Monitoring. Titanium customers also have application logs automatically sent to Anypoint Monitoring by default. In order for metrics and logs to be sent to Anypoint Monitoring, outbound network connectivity must be available. 
+
+If an outbound proxy is needed, configure Runtime Fabric to use a SOCKS5 proxy by following the instructions in xref:manage-proxy.adoc[Manage Proxies Used by Runtime Fabric].
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-monitor-applications.adoc]:
```suggestion
If you need an outbound proxy, configure Runtime Fabric to use a SOCKS5 proxy by following the instructions in xref:manage-proxy.adoc[Manage Proxies Used by Runtime Fabric].
```

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 9]
```diff
@@ -4,11 +4,25 @@ Applications and API gateways deployed to Runtime Fabric include native support
...
+
+To verify outbound network connectivity:
+
+. Ensure each node in Runtime Fabric can send outbound TCP (Lumberjack) communication using port 5044 to:
+.. US control plane: `dias-ingestor-nginx.prod.cloudhub.io`
+.. EU control plane: `dias-ingestor-nginx.prod-eu.msap.io`
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-monitor-applications.adoc]: To verify outbound network connectivity, ensure that each node in Runtime Fabric can send outbound TCP (Lumberjack) communication using port 5044 to:
* US control plane: `dias-ingestor-nginx.prod.cloudhub.io`
* EU control plane: `dias-ingestor-nginx.prod-eu.msap.io`

Comment (kbostrom) [reply modules/ROOT/pages/manage-monitor-applications.adoc]: Do not create a list or step for just one item.

File: modules/ROOT/pages/manage-monitor-applications.adoc [hunk 10]
```diff
@@ -4,11 +4,25 @@ Applications and API gateways deployed to Runtime Fabric include native support
...
+
+=== Disable Anypoint Monitoring for an application
```
Comment (kbostrom) [inline modules/ROOT/pages/manage-monitor-applications.adoc]:
```suggestion
=== Disable Anypoint Monitoring for an Application
```
...but why does this title specify "for an application" when the peer enabling title does not?

**Review by kbostrom — APPROVED** [review]
Here you go!

**Pull request merged**